code/modules/reagents/newchem/drugs.dm

```diff
@@ -457,4 +458,173 @@ datum/reagent/fliptonium/overdose_process(var/mob/living/M as mob)
 					M.drop_r_hand()
 					M.adjustToxLoss(1)
 	..()
-	return
\ No newline at end of file
+	return
+
+//////////////////////////////
+//		Synth-Drugs			//
+//////////////////////////////
+
+//Ultra-Lube: Meth
+/datum/reagent/lube/ultra
```

Comment:
Hilariously enough, making it a subtype of meth would actually result in slightly MORE copypasta.

The message changes/emote tweaks would require the same procs to be redone, and then I would need to also bring in the reaction_turf proc from lube.